src/partition/partitioner.cpp

```diff
@@ -174,14 +174,8 @@ int Partitioner::Run(const PartitionConfig &config)
     TIMER_STOP(packed_mlp);
     util::Log() << "MultiLevelPartition constructed in " << TIMER_SEC(packed_mlp) << " seconds";
 
-    TIMER_START(cell_storage);
```

Comment:
Sure, just forgot about heavy constructor. I will add a reader function for `CellStorage`